techmatters/terraso-mobile-client
# feat: spec compliance for ExternalLink

## Description

Reimplement `ExternalLink` to be compliant with spec -- to assist with future work, this involved spinning off a common `TextButton` component which supports this component as well as expected future work in the components documentation.

### Checklist

### Related Issues
#1289

### Verification steps

Open sections of the application containing external links. Verify that they match the component specification. (Note that this is not the same as `InternalLink`, which will eventually open an embedded web view rather than a browser window.)
